### PR TITLE
DDF-4441 Set date/time format in EndpointUtil

### PR DESCRIPTION
**Pull request by hayleynorton**
#### What does this PR do?
The GsonBuilder in EndpointUtil currently uses inconsistent time formats; this sets it to ISO
#### Who is reviewing it? 
@Bdthomson @andrewzimmer 
#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@bdeining
@djblue
#### How should this be tested?
- Create a workspace
- Look at the workspace network request
- Verify that the date formats match the second screenshot below, not the first
#### What are the relevant tickets?
[DDF-4441](https://codice.atlassian.net/browse/DDF-4441)
#### Screenshots
**BEFORE**
image
**AFTER**
image
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java [hunk 1]
```diff
@@ -136,6 +136,7 @@
           .disableHtmlEscaping()
           .serializeNulls()
           .registerTypeAdapterFactory(LongDoubleTypeAdapter.FACTORY)
+          .setDateFormat(ISO_8601_DATE_FORMAT)
```
Comment (paouelle) [inline catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java]: ❓ Shouldn't there be unit tests for this?

File: catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/util/EndpointUtilTest.java [hunk 2]
```diff
@@ -455,4 +455,13 @@ public void testParseDateWhiteSpaceString() {
     Instant dateConverted = endpointUtil.parseDate("  ");
     assertThat(dateConverted, nullValue());
   }
+
+  @Test
+  public void testGsonUsesIsoDateTimeFormat() {
```
Comment (Bdthomson) [inline catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/util/EndpointUtilTest.java]: ✏️ 	testGetJsonUsesIsoDateTimeFormat

Approved though.

**Comment by bdeining** [comment]
build now

**Comment by andrewzimmer** [comment]
Hero Successful 🎉 
- loaded workspace, verified correct date format 
screen shot 2019-01-17 at 9 08 13 am

**Comment by hayleynorton** [comment]
build now

**Pull request merged**